packages/file_selector/file_selector_web/integration_test/file_selector_web_test.dart

```diff
@@ -15,16 +13,18 @@ import 'package:file_selector_web/src/dom_helper.dart';
 void main() {
   group('FileSelectorWeb', () {
     IntegrationTestWidgetsFlutterBinding.ensureInitialized();
-    MockDomHelper mockDomHelper;
-    FileSelectorWeb plugin;
-
-    setUp(() {
-      mockDomHelper = MockDomHelper();
-      plugin = FileSelectorWeb(domHelper: mockDomHelper);
-    });
 
     group('openFile', () {
-      final mockFile = createXFile('1001', 'identity.png');
+      late MockDomHelper mockDomHelper;
+      late FileSelectorWeb plugin;
+      late XFile mockFile;
+
+      setUp(() {
```

Comment:
Setup that's specific to a single test, as you have now made both of these, should be in the test itself, not `setUp`.

Reply:
Removed